scripts/sanity_chk/sanitylib.py

```diff
@@ -3352,10 +3377,11 @@ def __init__(self):
         self.detected = []
         self.connected_hardware = []
 
-    def load_device_from_cmdline(self, serial, platform):
+    def load_device_from_cmdline(self, serial, platform, serial_pty):
```

Comment:
still do not like this signature:

load_device_from_cmdline(serial1, platform, serial2).

Please rearrange or use some other logic, ie. just use one serial parameter and decide whther it is pty or not in some other way(another parameter?)

Reply:
like this 61760148d8ac45b933b67dacd897a526ab5f9773 ?

Reply:
I will squash the fixup then